.github/workflows/relayrunner-pr.yml

```diff
@@ -0,0 +1,73 @@
+name: Build RelayRunner-PR
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, closed]
+
+    paths:
+      - 'Dockerfile'
+      - '**.js'
+      - '**.css'
+      - '.github/workflows/relayrunner-pr.yaml'
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_REPO: abassey2/relayrunner

```

Comment:
should you push this to retaildevcrews repo?

Reply:
There is no image on the retaildevcrews repo and I didn't have access to create the image. Could you create the image so I can build and push it to that repo?

Reply:
Where can I view the image and change it to public?

Reply:
https://github.com/orgs/retaildevcrews/packages

Reply:
hmm the image is not popping up for me. I am not sure what happened